Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs

```diff
@@ -155,16 +154,20 @@ private void InitActionsGrid()
             AvailableControlActionsGrid.SetTitleStyle((Style)TryFindResource("@ucTitleStyle_4"));
         }
 
-        private void AddToPageList()
-        {
-            ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
-            if (tempPage == null) App.PageList.Add(this);
-            else
-            {
-                App.PageList.Remove(tempPage);
-                App.PageList.Add(this);
-            }
-        }
+//private void AddToPageList()
+        //{
+        //    if (App.PageList != null)
+        //    {
+        //        ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
+        //        if (tempPage == null) App.PageList.Add(this);
+
+        //        else
+        //        {
+        //            App.PageList.Remove(tempPage);
+        //            App.PageList.Add(this);
+        //        }
+        //    }
+        //}
 
```

Comment:
delete the commented code or Codacy will mark it as issue

Reply:
commented code has been deleted.